testing/src/cypress/e2e/04-api-institution/02-instition-create-update-location.cy.ts

```diff
@@ -0,0 +1,103 @@
+import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
+import Authorization from "../../custom-command/common/authorization";
+import ApiData from "../data/endpoints/api-endpoints.json";
+
+const institutionHelperActions = new InstitutionHelperActions();
+
+function createOrUpdateInstitutionLocation(
+  token: string,
+  id: string,
+  canadian: boolean,
+  create: boolean
+) {
+  const uniqueId = id;
+  let body: object;
+  let method: string;
+  let url: string;
+  let status: number;
+  if (create) {
+    method = "POST";
+    url = `${ApiData.testEnv.baseUrl}${ApiData.endPoints.createInstitutionLocation}`;
+    status = 201;
+  } else {
+    method = "PATCH";
+    url = `${ApiData.testEnv.baseUrl}${ApiData.endPoints.createInstitutionLocation}/1`;
+    status = 200;
+  }
+  if (canadian) {
+    body = {
+      locationName: `Auto-${uniqueId}-location`,
+      institutionCode: "AUTO",
+      addressLine1: `Auto-${uniqueId}-Street`,
+      addressLine2: "14235",
+      selectedCountry: "Canada",
+      country: "Canada",
+      city: `Auto-${uniqueId}-City`,
+      canadaPostalCode: "A1A2A3",
+      primaryContactFirstName: `Auto-${uniqueId}-FirstName`,
+      primaryContactLastName: `Auto-${uniqueId}-LastName`,
+      primaryContactEmail: `Auto@${uniqueId}.com`,
+      primaryContactPhone: "32165496872",
+      submit: true,
```

Comment:
i don't think the is needed `submit: true,`

Reply:
upated